Classes/Facet/NumericRangeFacetRenderer.php

```diff
@@ -175,7 +175,6 @@ protected function loadStylesheets()
         if ($rangeFile !== '' && !$GLOBALS['TSFE']->additionalHeaderData['tx_solr-uiCss']) {
             $cssFile = GeneralUtility::createVersionNumberedFilename($GLOBALS['TSFE']->tmpl->getFileName($rangeFile));
             $GLOBALS['TSFE']->additionalHeaderData['tx_solr-uiCss'] = $rangeFile;
-            '<link href="' . $cssFile . '" rel="stylesheet" type="text/css" media="all" />';
```

Comment:
if it really is not used anymore in the templates, then the `$cssFile` variable can be removed, too.